### PR TITLE
Mixpanel kifi.com tweaks

### PR DESCRIPTION
**Pull request by stkem**
-Using “user” prefix for all events, just like the server
-Aliasing the external id, so we can track new signups end to end

### CODE REVIEW

File: kifi-backend/modules/shoebox/public/js/track.js [hunk 1]
```diff
@@ -75,7 +75,7 @@
   }
 
   function defaultViewHandler(path) {
-    mixpanel.track('viewed_internal_page',{
+    mixpanel.track('user_viewed_internal_page',{
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/js/track.js:78]: Out of curiosity, is the `user_` prefix helping to disambiguate something? If so, what?

Know how we’re going to distinguish website event names from extension ones yet?

Comment (stkem) [inline kifi-backend/modules/shoebox/public/js/track.js:78]: The server also has 'system' events so in order to distinguish them in mixpanel (they are simply in different repos on our end) we have this prefix. It's here purely for consistency with the server side events.
There is no prefix to distinguish site from extension events, but there is a spec which has all the specific event names.

Comment (2is10) [inline kifi-backend/modules/shoebox/public/js/track.js:78]: ​I see. I was thinking the associated user ID would make that clear, but
whatever.

I do think different prefixes for site vs extension vs iphone app would be
helpful.

Comment (stkem) [inline kifi-backend/modules/shoebox/public/js/track.js:78]: Honestly, I'm not really super happy with the naming here either, but I've given up the fight, if thats whats easiest to consume in mixpanel.

**Pull request merged**